upstash/redis/database/crud.go

```diff
@@ -0,0 +1,79 @@
+package database
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/upstash/terraform-provider-upstash/upstash/client"
+	"github.com/upstash/terraform-provider-upstash/upstash/utils"
+)
+
+func resourceDatabaseUpdate(ctx context.Context, data *schema.ResourceData, m interface{}) diag.Diagnostics {
+	c := m.(*client.UpstashClient)
+	databaseId := data.Get("database_id").(string)
+	if data.HasChange("multizone") {
+		if err := EnableMultiZone(c, databaseId, data.Get("multizone").(bool)); err != nil {
+			return diag.FromErr(err)
+		}
+	}
+	// I think this creates a problem. Doesnt give tls as a parameter for the client to use.
+	if data.HasChange("tls") {
```

Comment:
How does it work? If can user disable tls? What happens if they do?

Reply:
A forceNew validator is implemented for disabling ```tls``` and ```multizone``` in resources.go file. 
If the new attribute becomes false from true, resource is destroyed and the new one is created.
Also, added to docs that tls and multizone attributes would result in resource being destroyed and recreated, if that ever happens.